lib/instrumentation/index.js

```diff
@@ -38,10 +39,10 @@ var MODULES = [
   '@apollo/server',
   '@aws-sdk/smithy-client', // Instrument the base client which all AWS-SDK v3 clients extends
   ['@elastic/elasticsearch', '@elastic/elasticsearch-canary'],
-  '@node-redis/client/dist/lib/client',
-  '@node-redis/client/dist/lib/client/commands-queue',
   '@opentelemetry/api',
   '@opentelemetry/sdk-metrics',
+  '@node-redis/client/dist/lib/client',
```

Comment:
My move *breaks* alphabetical ordering, and puts the `@redis/...` and `@node-redis/...` modules together because they are all for instrumenting `redis@4`. Earlier versions of `redis@4` used a `@node-redis/...` org, then moved to `@redis/...`.    I don't care strongly for now.  Eventually I think `MODULES` will get restructured so there will only be one top-level `redis`-related entry.